test/integration/buildpod_admission_test.go

```diff
@@ -135,20 +135,28 @@ func TestBuildOverrideTolerations(t *testing.T) {
 		},
 	}
 
+	requiredTolerations := make(map[string]kapi.Toleration)
```

Comment:
Can you please change `tolerations` to `[]v1.Toleration` up above and build the map there?

Reply:
@soltysh updated